Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -882,7 +890,12 @@ public class SoomlaProfile
 		}
 
 		internal static void TryFireProfileInitialized () {
-			if (AllProvidersInitialized()) {
+			if (AllProvidersInitialized()
+#if !UNITY_EDITOR
+				&& nativeModulesInitialized
```

Comment:
`OnProfileInitialized` should be and will be fired if and only if:
1) all providers was ready;
2) Native part of SOOMLA Profile was initialized.